server/recruitment.py

```diff
@@ -89,24 +98,78 @@ def edit_applicant_notes(applicant_user_id, text):
         applicant.notes = text
         return {'status': 'ok'}
 
+def get_user_list():
+    # list of all Users with roles
+    result = {}
+    for user in User.query().run():
+        id = user.get_id()
+        user_name = Character.get(id).name if id else None
+        result[id] = {
+            'user_id': id,
+            'is_recruiter': user.is_recruiter,
+            'is_snr_recruiter': user.is_senior_recruiter,
+            'is_admin': user.is_admin,
+            'name': user_name,
+        }
+    return result
+
+def get_character_search_list(search_text):
+    # list of all chars with name beggining with search_text
+    result = {}
+    for character in Character.query().\
+            where(Character.name >= search_text).\
+            and_where(Character.name < search_text + u'\ufffd').\
+            run():
+        id = character.get_id()
+        result[id] = {
+            'user_id': id,
+            'name': character.name,
+        }
+    return result
 
-def get_applicant_list():
+def get_applicant_list(current_user):
+    current_user_id = current_user.get_id()
     return_list = []
     for applicant in User.query().\
             where(User.is_recruiter.is_false).\
             and_where(User.is_senior_recruiter.is_false).\
             and_where(User.is_admin.is_false).\
             run():
-        print(applicant)
+        applicant_id = applicant.get_id()
         recruiter_name = \
             Character.get(applicant.recruiter_id).name if applicant.recruiter_id else None
-        return_list.append({
-            'user_id': applicant.get_id(),
-            'recruiter_id': applicant.recruiter_id,
-            'recruiter_name': recruiter_name,
-            'status': applicant.status,
-            'name': applicant.name,
-        })
+        applicant_name = \
+            Character.get(applicant_id).name if applicant_id else None
+
+        print('applicant', applicant_name)
```

Comment:
I'll significantly condense this code later, but it should work! Good idea on the API change.

Reply:
I just dropped all the prints (6 lines saved)

I started with a long boolean expression, but it's harder to debug